analysis/tsne.py

```diff
@@ -337,8 +342,10 @@ def main(configs_yaml):
     dataloader = DataLoader(dataset.data, batch_size=batch_size, num_workers=num_workers, collate_fn=custom_collate)
     
     # Load model
-    if isOriginalVQGAN:
+    if isOriginalVQGAN=='VQGAN':
         model_type=VQModel
+    elif isOriginalVQGAN=='CW':
```

Comment:
before you check this in, make sure all yamls that use isOriginalVQGAN have been updated properly